content/browser-testing/browserstack.md

```diff
@@ -251,6 +253,8 @@ solutions by presenting the results in a more user friendly fashion.
 
 {{< a--close >}}
 
+{{< div--close >}}
```

Comment:
@davidmytton @tatticoder I stumbled on a few of these missing in the vendor reviews.

As a rule of thumb every shortcode suffixed with `--open` will need a closing one suffixed with `--close`. I understand these might be difficult to spot because they can be nested and markdown doesn't provide any indentation, I usually do a search across the page and match them manually.

To know whether you need an `a--close`, or `div-close` or else, just look into the `--open` one and see what's the starting tag. For instance `nav-wrapper--close` in this case [opens a `div`](https://github.com/consoledotdev/home/blob/fd4525158864e51a11b138916a37e3abf04e4361/themes/console-home/layouts/shortcodes/nav-wrapper--open.html#L1)

Reply:
I see there's quite a few others missing, tidying them all now.